webui/src/components/editrecord.jsx

```diff
@@ -622,19 +648,31 @@ const EditRecord = React.createClass({
         const text = this.state.waitingForServer ? "Updating record, please wait..." :
                       this.isForPublication() ? 'Save and Publish' :
                       this.state.dirty ? 'Save Draft' : 'The draft is up to date';
+        const future_doi = this.props.record.get('b2share').get('future_doi') || "";
```

Comment:
This causes a rendering bug if DOI creation is disabled.

Reply:
```suggestion
        const future_doi = this.props.record.get('b2share', Map()).get('future_doi', '') || "";
```

Reply:
Previous commit updated